docs/api/api.md

```diff
@@ -0,0 +1,359 @@
+| page | query | page number | No | integer |
+| page_size | query | page size | No | integer |
+| name | query | name of service | No | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | list response | [ [service](#service) ] |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/ssl
+
+#### GET
+##### Description:
+
+Return the SSL list according to the specified page number and page size, and can search by sni
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| page | query | page number | No | integer |
+| page_size | query | page size | No | integer |
+| sni | query | sni of SSL | No | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | list response | [ [ssl](#ssl) ] |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/upstreams
+
+#### GET
+##### Description:
+
+Return the upstream list according to the specified page number and page size, and can search by name
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| page | query | page number | No | integer |
+| page_size | query | page size | No | integer |
+| name | query | name of upstream | No | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | list response | [ [upstream](#upstream) ] |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### /apisix/admin/user/login
+
+#### POST
+##### Description:
+
+user login
+
+##### Parameters
+
+| Name | Located in | Description | Required | Schema |
+| ---- | ---------- | ----------- | -------- | ---- |
+| username | body | user name | Yes | string |
+| password | body | password | Yes | string |
+
+##### Responses
+
+| Code | Description | Schema |
+| ---- | ----------- | ------ |
+| 0 | login success | [ApiError](#ApiError) |
+| default | unexpected error | [ApiError](#ApiError) |
+
+### Models
+
+
+#### ApiError
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| code | long | response code | No |
+| message | string | response message | No |
+
+#### BaseInfo
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| create_time | long |  | No |
+| id | object |  | No |
+| update_time | long |  | No |
+
+#### Consumer
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| create_time | long |  | No |
+| desc | string |  | No |
+| id | object |  | No |
+| labels | object |  | No |
+| plugins | object |  | No |
+| update_time | long |  | No |
+| username | string |  | No |
+
+#### LoginInput
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| password | string | password | No |
+| username | string | user name | No |
+
+#### Route
```

Comment:
I think we can do this later.  

and we can add `SSL`, `Service`, `Upstream` later too